build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
   id "com.github.spotbugs" version "4.5.1"
 }
 
-version = '4.1.5-SNAPSHOT'
+version = '4.2.0-SNAPSHOT'
```

Comment:
We don't necessarily want to increment the version number yet.

Reply:
Sorry, but I suggested to do so at https://github.com/spotbugs/spotbugs/pull/1348#discussion_r515011220
I think this is new feature for SpotBugs, isn't it? 🤔